### PR TITLE
feat(content types): Don't return on content type the field for custo…

### PR DESCRIPTION
**Pull request by ioanpocol**
…m vocabulary with selection type 'do not show' (SDESK-3625)

### CODE REVIEW

File: apps/content_types/content_types.py [hunk 1]
```diff
@@ -253,12 +265,21 @@ def get_fields_map_and_names():
...
         fields_map[vocabulary.get('schema_field', vocabulary['_id'])] = vocabulary['_id']
         field_names[vocabulary['_id']] = vocabulary.get('display_name', vocabulary['_id'])
 
     return fields_map, field_names
 
 
+def get_forbiden_vocabularies():
+    vocabularies = get_resource_service('vocabularies').get_custom_vocabularies()
+
+    return (vocabulary for vocabulary in vocabularies
```
Comment (petrjasek) [inline apps/content_types/content_types.py]: imo we can filter it via db, that might be quicker than fetching all vocabularies with items

File: tests/content_types_test.py [hunk 2]
```diff
@@ -17,6 +17,21 @@ def find_one(self, req, **lookup):
...
 
 
+class MockVocabulariesService():
+
+    def get_custom_vocabularies(self):
+        return [
+            {
+                "_id": "test",
+                "field_type": None,
+                "selection_type": "do not show",
```
Comment (petrjasek) [inline tests/content_types_test.py]: seems like this is bool value, would be probably more efficient to store it as such.
also the name is a bit misleading imo, it seems like there is some selection type like radio/checkboxes/select for the field, while this controls if field is displayed

Comment (ioanpocol) [reply tests/content_types_test.py]: There are 3 values for selection_type: 'single selection', 'multi selection' and 'do not show'

Comment (petrjasek) [reply tests/content_types_test.py]: so what does it mean actually?

Comment (ioanpocol) [reply tests/content_types_test.py]: https://dev.sourcefabric.org/browse/SDESK-3551

if I select a "single value" type I expect the CV to be shown as a dropdown in the UI
if I select a "multiple value", I expect the CV to be shown as a multi-select UI component
Should disable the option to add to CPs the CVs that had set 'do not show' value

File: apps/content_types/content_types.py (hunk 1 above)
Comment (ioanpocol) [reply apps/content_types/content_types.py]: Done

File: tests/content_types_test.py (hunk 2 above)
Comment (petrjasek) [reply tests/content_types_test.py]: ok I see, if there should be single control in the cv editor for all these it makes sense. can you pls add the field to schema and list those as allowed values with some comment what it does?

Comment (ioanpocol) [reply tests/content_types_test.py]: It is already added to vocabularies schema:
https://github.com/superdesk/superdesk-core/blob/master/superdesk/vocabularies/vocabularies.py#L81

**Pull request merged**